Hoxtygen/next-fullstack
# feat get listings by user id(HOM-66)

## What does this PR do?
It gets listings based on logged-in user id

## Description of the task to be completed
* Fetch data from the API. 
* Create a reusable custom hook to get fetched data
* Create UI components to display fetched data.


## How should this be manually tested?
* Clone the repo
* Open a terminal on your PC and run `git clone <repo-url>`
* `cd` into the project directory and run `yarn install`
* Run `yarn dev` to start the server
* Open a browser and navigate to `http://localhost:3000/`
* Sign up as a new user or log in if you already have one.
* On the dashboard, navigate to the `New Listing` link and create as many as possible 
* Navigate to the `Listings` link to see created listings.


## What are the relevant Jira board stories
[HOM-66](https://wasiu-dev.atlassian.net/browse/HOM-66)


[HOM-66]: https://wasiu-dev.atlassian.net/browse/HOM-66?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
